python/vyos/nat.py

```diff
@@ -69,7 +69,10 @@ def parse_nat_rule(rule_conf, rule_id, nat_type, ipv6=False):
         else:
             translation_output.append('to')
             if addr:
-                translation_output.append(addr)
+                if not ipv6:
```

Comment:
An we use “bracketize” in the template?
For example https://github.com/vyos/vyos-1x/blob/f5e66a451dcd1d98be32e0a7800277cf81441e3d/data/templates/syslog/rsyslog.conf.j2#L32

Reply:
I mean for template
But it is to late :)